internal/payloads/reservation_payload.go

```diff
@@ -116,13 +134,24 @@ func NewNoopReservationResponse(reservation *models.NoopReservation) render.Rend
 func NewReservationListResponse(reservations []*models.Reservation) []render.Renderer {
 	list := make([]render.Renderer, 0, len(reservations))
 	for _, reservation := range reservations {
+		var finishedAt *time.Time
+		if reservation.FinishedAt.Valid {
+			finishedAt = &reservation.FinishedAt.Time
+		}
+		var success *bool
+		if reservation.Success.Valid {
+			success = &reservation.Success.Bool
+		}
```

Comment:
This hunk fixes `null` value for finished_at/success, it was not working properly.